www/front_src/src/Resources/Details/tabs/Details/Card.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { Paper, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  content: {
+    padding: theme.spacing(1, 2, 2, 2),
+  },
+}));
+
+interface Props {
+  children?: React.ReactNode;
+  className?: string;
+}
+
+const Card = ({ children, className }: Props): JSX.Element => {
```

Comment:
A `Card` component already exist in Material UI https://material-ui.com/components/cards/#card. I think it is beter to use it rather create a new one that display the same layout

Reply:
I know it does, I actually used it before this change, but it adds some extra spacing that I tried to reduce by coming up with our own one.